app/scripts/controllers/reviewRegistration.js

```diff
@@ -178,19 +178,29 @@ angular
         });
     };
 
-    $scope.editRegistrant = function(id) {
+    $scope.editRegistrant = registrantId => {
       $location
         .path(
           '/' +
             ($rootScope.registerMode || 'register') +
             '/' +
             conference.id +
             '/page/' +
-            conference.registrationPages[0].id,
+            conference.registrationPages.filter(page =>
+              $scope.pageIsVisible(page, registrantId),
+            )[0].id,
```

Comment:
The same bug would occur if you clicked edit on registrant but the page was empty, so this bug is resolved now and the user is correctly navigated to the first valid page.